packages/contracts/deploy/40_finalize-managing-dao/30_install-multisig-on-managing-dao.ts

```diff
@@ -68,6 +69,30 @@ const func: DeployFunction = async function (hre: EHRE) {
     `Prepared (Multisig: ${installationPreparedEvent.plugin}) to be applied on (ManagingDAO: ${managingDAOAddress}), see (tx: ${prepareTx.hash})`
   );
 
+  // Adding plugin to verify array
+  const multisigSetupAddress = await getContractAddress('MultisigSetup', hre);
+  const multisigSetup = await MultisigSetup__factory.connect(
+    multisigSetupAddress,
+    deployer
+  );
+  hre.aragonToVerfiyContracts.push({
+    address: installationPreparedEvent.plugin,
```

Comment:
do we need to verify this plugin in particular,
the implementation is already added to verify in `31_multisig_setup_conclude.ts`?

Reply:
Agreed but I added it just to be on the safe side. Especially this would verify the proxy and not the implementation. Depends a little on how OZ responds here https://forum.openzeppelin.com/t/unverified-erc1967-proxy-goerli/34875